src/libraries/pkg/baseline/packageIndex.json

```diff
@@ -6153,32 +6153,6 @@
         "4.0.1.0": "4.3.0"
       }
     },
-    "System.Security.Cryptography.OpenSsl": {
```

Comment:
Why removing entirely? We need to make sure this remains and any existing usages for downlevel frameworks use the past stable packages.

Reply:
We don't have any references to OpenSsl in non live built frameworks today. I wasn't sure if we want to leave or remove it. Will add it back.

Reply:
Please also add
```json
"InboxOn": {
  "net6.0": "6.0.0.0"
}
```

Reply:
done, thanks